scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -60,7 +60,7 @@ setTheme() {
 # Do NOT modify code below           #
 ######################################
 
-GLV_VERSION=v1.30.0
+GLV_VERSION=v1.29.1
```

Comment:
Restore the existing version. Although maybe increment to 1.30.1. I'll defer to @rdlrt or @Scitz0 on correct semver for gLiveView changes.